File: StraviaTec-Web/Models/CuentaBancaria.cs
using System;
using System.Collections.Generic;

namespace StraviaTec_Web.Models
{
    public partial class CuentaBancaria
    {
        public int Id { get; set; }
        public int IdCarrera { get; set; }
        public string Nombre { get; set; }

        public virtual Carrera IdCarreraNavigation { get; set; }
    }
}

File: StraviaTec-Web/ClientApp/src/app/models/cuentaBancaria.ts
export class CuentaBancaria {
    id: number;
    idCarrera: number;
    nombre: string;

    constructor(
        id: number,
        idCarrera: number,
        nombre: string) {
        this.id = id;
        this.idCarrera = idCarrera;
        this.nombre = nombre;
    }
}